tutorials/using-bucket-website-with-mkdocs/index.mdx

```diff
@@ -0,0 +1,380 @@
+
+        ```
+        ??? info
+
+            I am an information admonition. You can use me to give additional information to your user.
+        ```
+
+        Find out more about admonitions on the [MkDocs website](https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=admonitions#configuration).
+
+    ### Additional configuration
+
+        You can add the following additional configuration options in your `mkdocs.yml` file. The `features` component allows you to use various features of the Material theme.
+          ```
+          features:
+            - content.code.copy
+            # Adds a copy button to code blocks.
+            - navigation.sections
+            # Enables section-based navigation.
+            - navigation.path
+            # Shows the navigation path (breadcrumbs).
+            - navigation.top
+            # Adds a "back to top" button.
+            - search.suggest
+            # Enables search suggestions.
+          ```
+        Add the `plugins` component to define the MkDocs plugins you want to enable on your website. The `search` plugin allows you to add the search functionality and the `glightbox` plugin allows you to add images and videos.
+          ```
+          plugins:
+            # Lists the MkDocs plugins enabled for the site.
+            - search
+            # Adds search functionality.
+            - glightbox
+            # Adds lightbox functionality for images and videos.
+          ```
+        Add the copyright notice for your website.
+          ```
+          copyright: >
+            Copyright &copy; 2016 - 2024 Martin Donath
+          ```
+    ## Downloading the mkdocs file template
+
+        If you have only added the elements shown in the sections above, you can download the `mkdcos.yml` below.
+
+          <a href="/tutorials/using-bucket-website-with-mkdocs/mkdocs-example.yml" target="_blank" rel="noopener noreferrer" download="mkdocs-example.yml"> Download the mkdocs.yml file </a>
+
+</TabsTab>
+  <TabsTab label="Preview and deploy your MkDocs website">
+
+      Once your website's content is ready and configured, you can preview it to make sure that everything displays properly.
+
+      ## Previewing and generating a static website
+
+      1. Open the folder containing your `docs` directory in Visual Studio Code. The one we have created in this tutorial is named `my-folder`.
+      2. Open the Visual Studio Code terminal and paste the following command:
+          ```
+          mkdocs serve
+          ```
+      3. Copy the localhost link that displays in the output into your browser. Your website should display.
+      4. Check that everything on your website displays correctly and ensure that there are no broken links.
+      5. Run the following command in your terminal to generate your folder's content into a static website:
```

Comment:
Missing step: "control+c" to shut down local host, otherwise running `mkdocs build` in the same terminal won't work